3.7/Dockerfile

```diff
@@ -2,12 +2,13 @@
 
 # Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
```

Comment:
### Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template

Reply:
hey @cclauss, what are you referring to specifically in #124?

Reply:
That it edited individual Dockerfiles.

Reply:
i'm not sure which lines you're referring to specifically, but i don't believe these were edited manually. if there's an issue, i'm happy to resolve

Reply:
hi @cclauss i'm not sure which lines you're referring to specifically, but i don't believe these were edited manually. if there's an issue, i'm happy to resolve